types/src/block_metadata.rs

```diff
@@ -53,17 +50,21 @@ impl BlockMetadata {
         self.id
     }
 
-    pub fn into_inner(self) -> Result<(u64, u64, Vec<u8>, AccountAddress)> {
-        let vote_maps = lcs::to_bytes(&self.previous_block_votes)?;
-        Ok((self.round, self.timestamp_usecs, vote_maps, self.proposer))
+    pub fn into_inner(self) -> Result<(u64, u64, Vec<AccountAddress>, AccountAddress)> {
```

Comment:
why are we applying a result to this?

Reply:
ah good point, because lcs returns a result before but now we can remove it